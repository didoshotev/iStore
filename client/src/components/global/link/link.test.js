import renderer from 'react-test-renderer'
import TestingEvironment from '../../../test-utils/router'
import LinkComponnent from './link'



describe('LinkComponent', () => {

    it('renders content properly', () => {
        const tree = renderer.create(
            <TestingEvironment>
                <LinkComponnent href={'/'}>Test</LinkComponnent>
            </TestingEvironment>
        ).toJSON()
        console.log(tree);
        expect(tree).toMatchSnapshot()
    })
})