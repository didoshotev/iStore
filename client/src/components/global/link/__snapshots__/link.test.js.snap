// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkComponent renders content properly 1`] = `
<a
  className="a a--white"
  href="/"
  onClick={[Function]}
/>
`;
